sass/components/_timeperiod.scss

```diff
@@ -187,7 +187,7 @@
         top: 0px;
 
         > .cq-menu-btn {
-            padding: 16px 5px !important;
+            //padding: 16px 5px !important;
```

Comment:
@MahboobehMohammadi Would you plz remove the commented line and even the class name since it's not usable anymore.

Reply:
That was me :|

Reply:
@morteza-binary yes ashkan is working on this so i kept it so he knows what changes, he can remove comments after his done.